registrations/serializers.py

```diff
@@ -362,7 +365,57 @@ class Meta:
 
 class SignUpGroupSerializer(serializers.ModelSerializer):
     view_name = "signupgroup-detail"
-    signups = SignUpSerializer(many=True)
+
+    def get_fields(self):
+        fields = super().get_fields()
+        fields["signups"] = SignUpSerializer(
+            many=True, required=False, partial=self.partial
+        )
+        return fields
+
+    def validate(self, data):
+        validated_data = super().validate(data)
+
+        errors = {}
+
+        if (
+            "registration" in validated_data
+            and self.instance.registration != validated_data["registration"]
```

Comment:
To be on the safe side I think you should also check that `self.instance` exists i.e. is not `None`.
```suggestion
            and self.instance and self.instance.registration != validated_data["registration"]
```